packages-exp/analytics-compat/src/index.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { firebase } from '@firebase/app-compat';
+import { _FirebaseNamespace } from '@firebase/app-types/private';
```

Comment:
FYI, we may have merge conflict with https://github.com/firebase/firebase-js-sdk/pull/4461 depending on which one got merged first.